sockeye/coverage.py

```diff
@@ -77,8 +81,8 @@ class CountCoverage(Coverage):
     Coverage class that accumulates the attention weights for each source word.
     """
 
-    def __init__(self, prefix='') -> None:
-        self.prefix = prefix
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
why is it a good idea to not expose the prefix in all derived classes anymore? I would still expose it.